### ISSUE TITLE
docs(cv-adapter): static anchor spec + CV vendor confidence requirements — defines Phase 2 integration contract

### ISSUE THREAD

**Issue by yohei1126** [edgesentry/edgesentry-rs#344]
## Context

`docs/pipeline/cv-adapter-spec.md` currently documents EntityStream format (position, velocity, class). For Phase 2 (live camera), two additional pieces are required:

1. **Static anchor reporting** — CV vendor must periodically send known fixed-point observations so EdgeSentry can compute `drift_score` (#343)
2. **Calibrated confidence scores** — raw softmax is not acceptable; must be calibrated (Platt scaling or equivalent)

This issue documents the *interface requirements* so a CV vendor can integrate without waiting for EdgeSentry's internal implementation.

---

### Static anchor format

CV adapters must include an `anchors` field alongside `entities`:

```json
{
  "timestamp_ms": 1748600000000,
  "entities": [...],
  "anchors": [
    {
      "id": "floor-mark-A",
      "observed_x": 12.38,
      "observed_y": 8.71,
      "confidence": 0.97
    },
    {
      "id": "bollard-north",
      "observed_x": 45.12,
      "observed_y": 0.03,
      "confidence": 0.95
    }
  ]
}
```

Ground-truth positions are stored in `profile/params.toml`:

```toml
[[anchors]]
id            = "floor-mark-A"
world_x_m     = 12.40
world_y_m     = 8.70
description   = "Yellow painted cross, north aisle, 12m from gate"

[[anchors]]
id            = "bollard-north"
world_x_m     = 45.00
world_y_m     = 0.00
description   = "Steel bollard, quayside north"
```

EdgeSentry computes:
```
residual_i = ‖(observed_x_i, observed_y_i) − (world_x_i, world_y_i)‖
drift_score = mean(residual_i)
```

**Minimum:** 3 anchors. **Recommended:** 4+ anchors covering the scene corners.

### Anchor reporting frequency

Anchors must be reported at least once every 30 seconds. Missing anchor reports for > 60 seconds triggers `CoverageGapReason::AnchorDriftDetected`.

### Confidence score requirements

| Requirement | Acceptable | Not acceptable |
|---|---|---|
| Source | Calibrated probability (Platt scaling, isotonic regression) | Raw softmax output |
| Range | 0.0–1.0 (true probability) | Uncalibrated logit scores |
| Meaning | P(correct detection) | Arbitrary ranking score |
| Validation | Must be tested against held-out data; ECE < 0.05 | Untested |

**Why:** Raw softmax scores from YOLO and similar models are systematically overconfident (e.g., 0.95 confidence for 70% actual accuracy). Using them directly would inflate `S_cv` and produce false `Certified` evidence quality ratings.

### Recalibration reporting

When the CV adapter detects it cannot reliably observe the scene, it should explicitly emit a coverage gap:

```json
{
  "timestamp_ms": 1748600000000,
  "entities": [],
  "anchors": [],
  "coverage_gap": {
    "reason": "low_confidence",
    "confidence_mean": 0.31,
    "message": "Lens fog detected — all detections below 0.50 threshold"
  }
}
```

---

## Relationship to other issues

- #341 (physics consistency check) — EdgeSentry-side validation; no CV vendor needed
- #342 (CoverageGap event) — EdgeSentry seals gap events into audit chain
- #343 (dual confidence score) — uses anchor data and CV confidence together
- #336 (edgesentry-calibrate) — homography math for pixel→metre conversion

---

## What EdgeSentry validates vs what CV vendor is responsible for

| Concern | CV vendor | EdgeSentry |
|---|---|---|
| Pixel → metre accuracy < 0.5m | ✓ (calibration) | Verifies via anchor residuals |
| Object detection accuracy | ✓ | Cannot verify without ground truth |
| Confidence calibration quality | ✓ | Trusts the value; flags if consistently low |
| Scene coverage continuity | ✓ (explicit gap reports) | Detects via packet timeout |
| Physical plausibility of output | — | ✓ physics consistency check (#341) |

**Priority:** P1 — this document defines what CV vendors must implement before Phase 2 integration can begin.
